municipalities.md

```diff
@@ -13,37 +13,37 @@ permalink: /municipalities
+The current information architecture makes it difficult for municipalities to securely exchange data in their own backoffice or between other municipalities. The current systems are lacking the flexibility to quickly adept and innovate in order to implement rapid change in public policy and serve the changing needs of its citizens. This makes maintaining and altering exisiting software increasingly slow and expensive resulting in a lower quality of service with higher costs. OpenZaak was developed to solve this problem helping municipalities increase their level of service and lower the cost of ownership.
 
 ### Background
 
-OpenZaak itself is not a case management or document management solution but rather a ready-to-use backend solution which functions as a directory of case registration including meta data describing the case. OpenZaak implements the national API standards for case-oriented working ([ZGW API's](https://www.vngrealisatie.nl/producten/api-standaarden-zaakgericht-werken)) developed by [VNG Realisatie](https://www.vngrealisatie.nl/). In order to fully utilize OpenZaak it needs to connect with other systems or components. Document management systems, or specialized systems to handle permits as well as the case handling component, MyApp or apps for MOR reports are examples of such systems and components.
+OpenZaak itself is not a case management or document management solution but rather a ready-to-use backend solution which functions as a directory of case registration including metadata describing the case. OpenZaak implements the national API standards for case-oriented working ([ZGW API's](https://www.vngrealisatie.nl/producten/api-standaarden-zaakgericht-werken)) developed by [VNG Realisatie](https://www.vngrealisatie.nl/). In order to fully utilize OpenZaak it needs to connect with other systems or components. Document management systems, or specialized systems to handle permits as well as the case handling component, MyApp or apps for MOR reports are examples of such systems and components.
 
-OpenZaak originally was developed by Maykin Media B.V. commissioned by Amsterdam, Rotterdam, Utrecht, Tilburg, Arnhem, Haarlem, 's-Hertogenbosch, Delft and a coalition of Hoorn, Medemblik, Stede Broec, Drechteland, Enkhuizen (SED), directed by [Dimpact](https://www.dimpact.nl/).
+OpenZaak was originally developed by Maykin Media B.V. and commissioned by Amsterdam, Rotterdam, Utrecht, Tilburg, Arnhem, Haarlem, 's-Hertogenbosch, Delft and a coalition of Hoorn, Medemblik, Stede Broec, Drechteland, Enkhuizen (SED) under direction of [Dimpact](https://www.dimpact.nl/).
```

Comment:
I might choose "initially" rather than "originally", but either is okay.

Reply:
I like initially, I'll change that.